src/Lib/Filters/Filter.php

```diff
@@ -91,7 +91,7 @@ public function setFilters($filters = [])
      *
      * @param string $operator the filter operator (eq,ne etc)
      * @param string $field the field to filter on
-     * @param string $value the value of the attribute to operate on
+     * @param string|mixed $value the value of the attribute to operate on
```

Comment:
Could you change ordering of fields according to ordering params in function?
I guess, sting|array is a correct one, isn't it?

Reply:
1. ordering was changed
2. agreed, it will be more strict